src/azure/cli/commands/parameters.py

```diff
@@ -73,6 +73,19 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
 def get_enum_choices(enum_type):
     return [x.value for x in enum_type]
 
+class IgnoreAction(argparse.Action): # pylint: disable=too-few-public-methods
+    def __call__(self, parser, namespace, values, option_string=None):
+        raise argparse.ArgumentError(None, 'unrecognized argument: {} {}'.format(
+            option_string, values or ''))
+
+# CONSTANTS
+
+IGNORE_TYPE = CliArgumentType(
```

Comment:
SUPPRESS is a constant though and this is a class instance, so I think we'd be more inline with Python styles if we renamed it and its usages to lowercase

---
In reply to: [76151727](https://github.com/Azure/azure-cli/pull/744#discussion_r76151727) [](ancestors = 76151727,76151354)